lib/elixir/lib/agent.ex

```diff
@@ -150,6 +150,13 @@ defmodule Agent do
 
   If the given function callback fails with `reason`, the function returns
   `{:error, reason}`.
+
+  ## Examples
+
+      iex> {:ok, pid} = Agent.start_link fn -> 42 end
```

Comment:
Let's have parenthesis here and in all other places, we usually omit them only in DSL and macro calls.